src/main/java/com/kelvinconnect/discord/command/stando/StandoCommand.java

```diff
@@ -25,31 +25,29 @@
     private static final String LEARN_MEDIUM = "The Times said ";
     private static final String LEARN_HIGH = "The Daily Mail said ";
 
+    private static final String QUERY_COUNT = "How much do you know?";
+    private static final String QUERY_DRINK = "What do you drink?";
+
+
+    private static final String[] drinkEmojis = {"\uD83C\uDF7A", "\uD83C\uDF7B", "\uD83C\uDF77", "\uD83C\uDF78", "\uD83C\uDF79",
```

Comment:
May as well, while we're here, add the "clinking champagne glasses" and "whisky" emojis. Looks like those are
```
"\uD83E\uDD42"
"\uD83E\uDD43"
```

Also ALL_CAPS for `static final` fields :)